modules/staking/utils_staking_pool.go

```diff
@@ -12,5 +13,35 @@ func (m *Module) GetStakingPool(height int64) (*types.Pool, error) {
 		return nil, fmt.Errorf("error while getting staking pool: %s", err)
 	}
 
-	return types.NewPool(pool.BondedTokens, pool.NotBondedTokens, height), nil
+	validatorsList, err := m.db.GetValidators()
+	if err != nil {
+		return nil, fmt.Errorf("error while getting validators list: %s", err)
+	}
+
+	var unbondingTokens int64
+	for _, validator := range validatorsList {
+		unbondingDelegations, err := m.source.GetUnbondingDelegationsFromValidator(
+			height,
+			validator.GetOperator(),
+			nil,
+		)
```

Comment:
I think it's good if we can check all the pages from the response we adding a method like this

```go
func (m *Module) getTotalUnbondingDelegationsFromValidator(height int64, valOperAddr string) []stakingtypes.UnbondingDelegation {
	var unbondingDelegations []stakingtypes.UnbondingDelegation

	var nextKey []byte
	var stop = false
	for !stop {
		res, err := m.source.GetUnbondingDelegationsFromValidator(
			height,
			valOperAddr,
			&query.PageRequest{Key: nextKey},
		)
		if err != nil {
			continue
		}

		unbondingDelegations = append(unbondingDelegations, res.UnbondingResponses...)

		nextKey = res.Pagination.NextKey
		stop = len(res.Pagination.NextKey) == 0
	}

	return unbondingDelegations
}
```